index.js

```diff
@@ -8,16 +8,33 @@ function ellipsize(string, desiredLength) {
   }
 }
 
-module.exports = function printSlowNodes(tree, factor) {
+function getLargestString(strings) {
+  return Math.max(...(strings.map(string => string.length)));
```

Comment:
😭 we haven't dropped old node (still support Node 0.12 here)

CI fails here:

```
/home/travis/build/rwjblue/broccoli-slow-trees/index.js:12
  return Math.max(...(strings.map(string => string.length)));
                  ^
SyntaxError: Unexpected token .
```

```suggestion
  return Math.max.apply(Math, strings.map(string => string.length));
```